packages/react/src/file-input/docs/overview.mdx

```diff
@@ -70,3 +71,25 @@ Use the `invalid` prop to indicate if the user input is invalid.
 ```jsx live
 <FileInput label="Invalid" invalid message="This input is invalid" />
 ```
+
+## Accept
+
+Use the `accept` prop to restrict the accepted file types. This will automatically add a hint to guide users, although you can override this by providing your own `hint`.
+
+_NOTE: for security reasons you must validate the file type in your backend and not rely on frontend validation._
+
+### Accept with default hint
+
+```jsx live
+<FileInput accept={'image/jpg, image/jpeg, image/png'} label="Upload image" />
+```
+
+### Accept with custom hint
+
+```jsx live
+<FileInput
+	accept={'image/jpg, image/jpeg, image/png'}
+	hint="Only upload image files of type .jpg, .jpeg, .png"
```

Comment:
Perhaps?
```suggestion
	hint="Valid file types: .jpg, .jpeg, .png"
```

Reply:
Perhaps that should be the default? Hmmmm

Reply:
I deliberately used language that was contrasted with the default, so that it's obvious. i.e. no `:`, different length.

Reply:
Hmmm. I wonder if the example text alludes to something we condone though? And I don't think we'd condone "Only upload…". Perhaps you could expand the hint as well, talking about filesize or something to make it clear it's different?

Reply:
We don't seem to have file size so I've gone for max files. Let me know what you think.

- [ ] update hint